mv_to_campaign.csh

```diff
@@ -32,7 +32,10 @@
 #   https://www2.cisl.ucar.edu/sites/default/files/CISL_GlobusCLI_Nov2018.html
 
 # Get CASE environment variables from the central variables file.
+cd /glade/work/raeder/Exp/f.e21.FHIST_BGC.f09_025.CAM6assim.011
 source ./data_scripts.csh
+cd -
```

Comment:
You could just 

source /glade/work/raeder/Exp/f.e21.FHIST_BGC.f09_025.CAM6assim.011/data_scripts.csh

Reply:
i agree that cd'ing over and back instead of running a single script is not as straightforward as sourcing the file.  data_scripts.csh should be completely self-contained and shouldn't reference other things in the current directory, so it can be sourced from anywhere.

Reply:
Since data_scripts.csh runs xmlquery it must be run in $CASEROOT.

Reply:
longer term, if xmlquery is really needed, then inside the param script i would have put a test for the current dir, and cd'd into the case dir inside the param script, so you don't have to do this outside of each place you source this script.  a param script is pretty standard and it's very unusual that it has to be run from a particular place.  so i'd put that inside the script so from the outside it looks more standard.